vassal-doc/src/main/readme-referencemanual/ReferenceManual/HexGrid.adoc

```diff
@@ -7,35 +7,38 @@
 
 === Hex Grid
 
-[cols=",",]
-|===
-|A standard hexagonal grid for regulating movement on a <<Board.adoc#top,Board>>. The default hexagonal grid layout is "Flat Topped".
+A standard hexagonal grid for regulating movement on a <<Board.adoc#top,Board>>. The default hexagonal grid layout is "Flat Topped".
 
-*Sideways:*   Check this box to make the hexrows of the grid run right-to-left instead of top-to-bottom.
+[width="100%",cols="50%a,50%"a,]
+|===
+|
+*Sideways (hexrows go horizontal):*::   Check this box to make the hexrows of the grid run right-to-left instead of top-to-bottom.
 This changes the hexagonal grid layout from "Flat Topped" to "Pointy Topped".
-(Setting the grid to be sideways switches the meanings of horizontal/vertical and x/y below).
 
-*X,Y offset:*  The horizontal and vertical position of the center of the first hex of the grid.
+NOTE: (Setting the grid to be sideways switches the meanings of horizontal/vertical and x/y below).
```

Comment:
Does this also switch the meaning of Height & Width?  It's not clear based on the current wording....

Reply:
Agreed. Have fixed it. This is what the NOTE: keyword generates:

![image](https://user-images.githubusercontent.com/1622667/130602482-19fffa91-33e3-4f05-9788-41f7f0ad3a6c.png)